nemo/collections/asr/modules/rnnt.py

```diff
@@ -1127,3 +1183,11 @@ def forward(self, encoder_outputs, decoder_inputs, decoder_lengths, state_h, sta
         state_h, state_c = decoder_outputs[2][0], decoder_outputs[2][1]
         joint_output = self.joint(encoder_outputs, decoder_output)
         return (joint_output, decoder_length, state_h, state_c)
+
+
+# Add the adapter compatible modules to the registry
+if adapter_mixins.get_registered_adapter(RNNTDecoder) is None:
```

Comment:
Is there a way to have the list of these three parts in a list and then a loop over them instead of separate block for each one?

Reply:
Yes possible.

Reply:
Done